lib/aadutils.js

```diff
@@ -22,6 +22,7 @@
  */
 'use strict';
 
+const base64url = require('base64url');
 const crypto = require('crypto');
```

Comment:
Is this the crypto library: https://github.com/Gozala/crypto? That repo does not look like supported at all. I know you did not add this but am not sure if we want to refer to this library.

npm link :https://www.npmjs.com/package/crypto